portality/view/account.py

```diff
@@ -308,33 +309,40 @@ def logout():
 class RegisterForm(RedirectForm):
     identifier = StringField('ID', [ReservedUsernames(), IdAvailable()])
     name = StringField('Name', [validators.Optional(), validators.Length(min=3, max=64)])
-    email = StringField('Email address', [
+    sender_email = StringField('Email address', [
         validators.DataRequired(),
         validators.Length(min=3, max=254),
         validators.Email(message='Must be a valid email address'),
         EmailAvailable(message="That email address is already in use. Please <a href='/account/forgot'>reset your password</a>. If you still cannot login, <a href='/contact'>contact us</a>.")
     ])
     roles = StringField('Roles')
-    recaptcha_value = HiddenField()
+    # These are honeypot (bot-trap) fields
+    email = StringField('email')
+    hptimer = IntegerField('hptimer')
```

Comment:
should be DecimalField ?

Reply:
![image](https://github.com/user-attachments/assets/96d680dc-6e83-4adf-af1d-7e5e82c88807)